components/trial-bar/dropdown.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import Dropdown from '~/components/menu-dropdown'; // `~` is replaced with design-system-react at runtime
+import DropdownTrigger from '~/components/menu-dropdown/button-trigger';
+import MenuDropdown from '~/components/menu-dropdown/menu-dropdown';
+import Button from '~/components/button';
+
+import { TRIAL_BAR_DROPDOWN } from '../../utilities/constants';
+
+// This component accepts the same props as MenuDropdown.
+// eslint-disable-next-line react/forbid-foreign-prop-types
+const { propTypes } = MenuDropdown;
+
+const TrialBarDropdown = (props) => {
+	const { label, ...rest } = props;
+	return (
+		<Dropdown {...rest} inverse>
```

Comment:
Is it really necessary to add a new prop `inverse` on dropdown component since it's only adding a class name? Maybe we can just use `className` instead?

Reply:
We can use just the `className` but isn't having an `inverse` variant of the dropdown worth it?